worker/containerd/containerd.go

```diff
@@ -48,28 +47,13 @@ type WorkerOptions struct {
 }
 
 // NewWorkerOpt creates a WorkerOpt.
-func NewWorkerOpt(
-	workerOpts WorkerOptions,
-	opts ...containerd.ClientOpt,
-) (base.WorkerOpt, error) {
+func NewWorkerOpt(workerOpts WorkerOptions, opts ...containerd.ClientOpt) (base.WorkerOpt, error) {
 	opts = append(opts, containerd.WithDefaultNamespace(workerOpts.Namespace))
-
-	address := workerOpts.Address
-
-	if goRuntime.GOOS == "windows" {
-		// TODO(profnandaa): once the upstream PR[1] is merged and
```

Comment:
thanks for helping with this TODO!

Reply:
YW! I stumbled on it while working on some things in moby/moby, and it was low-hanging fruit 🥹😅